src/navigation/navigation.cc

```diff
@@ -124,25 +119,18 @@ float Navigation::limitVelocity(float vel) {
 	return          std::max({new_vel, robot_vel_[0] + min_accel_ * dt_, min_vel_});
 }
 
-// Move forward a set amount in a straight line
-void Navigation::moveForwards(float start, float dist){
-	// Minimum distance where car reaches max velocity in the middle
-	float min_dist = accel_dist_ + decel_dist_;
-
-	// Solve for distance at which to start decelerating
-	float inflection_dist;
-	if (dist >= min_dist) {
-		// Normal operation: speed up, cruise at that velocity, and then slow down
-		inflection_dist = dist - decel_dist_;
-	}else{
-		// Car does not reach max velocity, inflection point is interpolated
-		inflection_dist = dist*(max_accel_/(max_accel_-min_accel_));
-	}
+// Move forward a set distance in a straight line
+void Navigation::moveForwards(Vector2f& start, float dist){
+	// Update how far you've come and how far to go
+	float dist_traveled = (robot_loc_ - start).norm();
+	float dist_to_go = dist - dist_traveled;
```

Comment:
This is just straight line distance, but our car is gonna drive along the full arc length. tbh it's probably a decent approximation but I think the more "correct" way is to use the arc length